src/plugins/regexdispatcher/README.md

```diff
@@ -0,0 +1,36 @@
+- infos = Information about the regexdispatcher plugin is in keys below
+- infos/author = e1528532 <e1528532@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = 
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained experimental
+- infos/metadata =
+- infos/description = a plugin which generates regex-representations of specification keywords
+
+## Introduction
+
+A plugin which generates regex-representations of specification keywords for the 
+typechecker plugin before saving a configuration specification. Intended to be 
+mounted before the typechecker plugin.
```

Comment:
"before" is misleading: the actual order need to be specified with placements or infos/ordering